hera_cal/tests/test_firstcal.py

```diff
@@ -313,3 +313,21 @@ def test_single_file_execution_nocalfile(self):
         nt.assert_true(os.path.exists(objective_file))
         os.remove(objective_file)
         return
+
+    def test_overwrite(self):
+        objective_file = os.path.join(
+            DATA_PATH, 'zen.2457698.40355.xx.HH.uvcAA.first.calfits')
+        xx_vis4real = os.path.join(DATA_PATH, xx_vis)
+        if os.path.exists(objective_file):
+            os.remove(objective_file)
+        _ = open(objective_file, 'a').close()
+        o = firstcal.firstcal_option_parser()
+        cmd = "-C {0} -p xx --ex_ants=81 --overwrite {1}".format(calfile, xx_vis4real)
+        opts, files = o.parse_args(cmd.split())
+        history = 'history'
+        firstcal.firstcal_run(files, opts, history)
+        uvc = UVCal()
+        uvc.read_calfits(objective_file)
```

Comment:
Great fix! Could you just add a couple of simple assert statements here? I'm thinking along the lines of `nt.assert_equal(uvc.Nants_data, 18)` (or whatever it's supposed to be for this file). That way we can make sure the metadata is being set properly.

Reply:
Hey, yeah I added a metadata test for the test_overwrite() function. It seems however that those kinds of tests really belong in the test_singlefile_execution() test, b/c that is directly looking to test that sort of thing. I think the test_overwrite() function should look to specifically target the additional functionality brought by the --overwrite option, which is to simply write out the file even if the file already exists.

Reply:
I totally agree, and I think we should have more robust tests in the main test as well. My concern was more with just adding an assert statement for nose to find, since I've seen instances of nose skipping tests if an assert statement is missing. I appreciate the addition of the assert statement.